src/esp/physics/PhysicsManager.cpp

```diff
@@ -224,6 +224,8 @@ int PhysicsManager::addObject(
     resourceManager_.addObjectToDrawables(obj->getInitializationAttributes(),
                                           obj->visualNode_, drawables,
                                           obj->visualNodes_, lightSetup);
+    // Set the scalingNode_ to the first child of the visualNode_
+    obj->scalingNode_ = obj->visualNodes_[1];
```

Comment:
Can you add some kind of assert that `visualNodes_[1]` is actually what you're expecting here? E.g. in case upstream code changes.